From 3ea6a7423b9f7b71bff50425d1bbad55eeb140df Mon Sep 17 00:00:00 2001
From: Cristi Mitrana <mitcr01@mitcr01.solvit.ro>
Date: Fri, 14 Sep 2018 13:29:28 +0300
Subject: [PATCH] Fix the `opacity` for the hold-to-skip message.

This fixed the bug when toggling the visibility by calling `setOpacity` on the TextComponent
didn't have any effect.
---
 es-core/src/guis/GuiInputConfig.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
 mode change 100644 => 100755 es-core/src/guis/GuiInputConfig.cpp

diff --git a/es-core/src/guis/GuiInputConfig.cpp b/es-core/src/guis/GuiInputConfig.cpp
old mode 100644
new mode 100755
index 934551a66..fe3b95e9d
--- a/es-core/src/guis/GuiInputConfig.cpp
+++ b/es-core/src/guis/GuiInputConfig.cpp
@@ -81,7 +81,7 @@ GuiInputConfig::GuiInputConfig(Window* window, InputConfig* target, bool reconfi
 	mSubtitle1 = std::make_shared<TextComponent>(mWindow, Utils::String::toUpper(ss.str()), Font::get(FONT_SIZE_MEDIUM), 0x555555FF, ALIGN_CENTER);
 	mGrid.setEntry(mSubtitle1, Vector2i(0, 2), false, true);
 
-	mSubtitle2 = std::make_shared<TextComponent>(mWindow, "HOLD ANY BUTTON TO SKIP", Font::get(FONT_SIZE_SMALL), 0x99999900, ALIGN_CENTER);
+	mSubtitle2 = std::make_shared<TextComponent>(mWindow, "HOLD ANY BUTTON TO SKIP", Font::get(FONT_SIZE_SMALL), 0x999999FF, ALIGN_CENTER);
 	mGrid.setEntry(mSubtitle2, Vector2i(0, 3), false, true);
 
 	// 4 is a spacer row
